exporters/otlp/test/otlp_grpc_exporter_test.cc

```diff
@@ -17,17 +17,23 @@
 // That is because `std::result_of` has been removed in C++20.
 
 #  include "opentelemetry/exporters/otlp/otlp_grpc_exporter.h"
-
+#  include "opentelemetry/exporters/otlp/otlp_grpc_exporter_factory.h"
 #  include "opentelemetry/exporters/otlp/protobuf_include_prefix.h"
+#  include "opentelemetry/nostd/shared_ptr.h"
```

Comment:
Move nostd/shared_ptr.h outside of the protobuf include prefix + suffix block.

Reply:
I never bothered to notice that these includes carried special meaning. Good to know!